data/sql_updates/totals_combined.sql

```diff
@@ -54,11 +58,15 @@ first_subset as (
     from disclosure.v_sum_and_det_sum_report
     where
         get_cycle(rpt_yr) >= :START_YEAR
+        -- issue #2601: F5 has both regular and 24/48 notices data included in disclosure.v_sum_and_det_sum_report, need to exclude the 24/48 hours notice data
+        and (form_tp_cd != 'F5'or (form_tp_cd = 'F5' and rpt_tp not in ('24','48')))
+        --
     order by
         cmte_id,
         get_cycle(rpt_yr),
         to_timestamp(cvg_end_dt) asc
 ),
+*/
```

Comment:
One other thing I forgot to note the other day - if this chunk of code is no longer referenced anywhere, we can just remove it instead of commenting it out.  This keeps things clean and if we do ever need it back, it is in our source control history. :-)